### PR TITLE
[GLUTEN-4917][VL] Enable celeborn test in CI

### PR DESCRIPTION
**Pull request by PHILO-HE**
## What changes were proposed in this pull request?

Enable celeborn test in CI.

## How was this patch tested?

N/A

### CODE REVIEW

File: .github/workflows/velox_docker.yml [hunk 1]
```diff
@@ -284,6 +284,52 @@ jobs:
...
+          bash -c "echo -e 'CELEBORN_MASTER_MEMORY=4g\nCELEBORN_WORKER_MEMORY=4g\nCELEBORN_WORKER_OFFHEAP_MEMORY=8g' > ./conf/celeborn-env.sh" && \
+          bash -c "echo -e 'celeborn.worker.commitFiles.threads 128\nceleborn.worker.sortPartition.threads 64' > ./conf/celeborn-defaults.conf" && \
+          bash ./sbin/start-master.sh && bash ./sbin/start-worker.sh && \
+          cd $GITHUB_WORKSPACE/tools/gluten-it && mvn clean install -Pspark-3.2,rss ${EXTRA_PROFILE} && \
+          GLUTEN_IT_JVM_ARGS=-Xmx5G sbin/gluten-it.sh queries-compare \
+            --local --preset=velox-with-celeborn --benchmark-type=h --error-on-memleak --off-heap-size=10g -s=0.1 --threads=8 --iterations=1 && \
+          GLUTEN_IT_JVM_ARGS=-Xmx5G sbin/gluten-it.sh queries-compare \
+            --local --preset=velox-with-celeborn --benchmark-type=ds --error-on-memleak --off-heap-size=10g -s=0.1 --threads=8 --iterations=1 && \
+          bash /opt/celeborn/sbin/stop-worker.sh && \
+          bash /opt/celeborn/sbin/stop-master.sh && rm -rf /opt/celeborn
```
Comment (zhouyuan) [inline .github/workflows/velox_docker.yml:331]: can we ignore the stop error issue first?

Comment (PHILO-HE) [reply .github/workflows/velox_docker.yml]: Just removed the stop commands. Thanks!

Comment (PHILO-HE) [reply .github/workflows/velox_docker.yml]: @kerwin-zk, they are executed in different containers. So I believe there is no conflict issue.

It's still strange that stopping command needs long wait time, but all queries have been executed successfully. Even though a smaller SF (0.1) is used, this issue still exists. I cannot reproduce it in my local docker container.

**Review by zhouyuan — APPROVED** [review]
👍

**Review by kerwin-zk — APPROVED** [review]

**Pull request merged**